raiden/tests/utils/mock_client.py

```diff
@@ -421,9 +418,9 @@ def settled(self):
         return self.contract.settled
 
     def detail(self, our_address):
-        partner_address = self.contract.partner(our_address)
-
         our_balance = self.contract.participants[our_address].deposit
+
+        partner_address = self.contract.partner(our_address)
```

Comment:
Here `contract` is a python class that simulates the netting channel behavior, `partner` is just a helper function from this python implementation that returns the partner address (and the reason for why the solidity implementation ended up having a method of the same name).